src/Microsoft.Health.Dicom.Core/DicomCoreResource.resx

```diff
@@ -335,4 +335,12 @@ The first part date {1} should be lesser than or equal to the second part date {
     <value>Value length is not {0}.</value>
     <comment>{0} required length.</comment>
   </data>
+  <data name="InvalidPrivateCreator" xml:space="preserve">
```

Comment:
Can we be more specific here?
PrivateCreatorNotNull
Private creator is a string set to represent the creator of a private tag. Since the tag being submitted is a standard tag private creator is expected to be empty.

Reply:
Will fix it.

Reply:
how about PrivateCreatorNotEmpty?